php-nginx/composer.sh

```diff
@@ -66,9 +67,13 @@ EOF
     # no need for the token to stay around in the env
     unset COMPOSER_GITHUB_OAUTH_TOKEN
 
+    # Workaround for https://github.com/docker/docker/issues/6047
+    # We want to remove when Container Builder starts to use newer Docker.
+    rm -rf ${APP_DIR}/vendor
```

Comment:
I really don't like force-removing the vendor directory. When will Container Builder be updated?